libr/core/cmd.c

```diff
@@ -1456,14 +1456,13 @@ static int cmd_kuery(void *data, const char *input) {
 			}
 			cur_cmd = r_str_ndup (out, cur_pos - out);
 
-			r_cons_printf ("\n\n\"%s\" : [", cur_cmd);
+			r_cons_printf ("\n\"%s\" : [", cur_cmd);
```

Comment:
ideally this whole function shuold be using the PJ API instead of constructing this with unfiltered strings

Reply:
I would love to, but I currently don't have enough knowledge on that.
Maybe you can open an issue and we can work on it later?

Reply:
the PJ api is very simple to use, and the new development model is commited to follow all the guidelines in order to merge stuff. you can have a look at how to use PJ in this commit 0d701a3b7972ced7eb450504ffbdffbe8273c303, but there are several other examples out there.. in short:

```c
PJ * pj = pj_new ();
pj_o (pj);
pj_ko (pj, "anal");
pj_ka (pj, "cur_cmd");
foreach(cmd) pj_s( pj, cmd);
pj_end (pj);
pj_end(pj);
pj_end (pj);
char *s = pj_drain(pj);
r_cons_printf ("%s%c", s, 10); // my keyboard doesnt have backslash)
free (s);

Reply:
Thank you, I'm looking into it.

Reply:
Cool! let me know if you have any question

Reply:
Can you explain a bit about `foreach(cmd) pj_s( pj, cmd);` part?
Sorry for taking a lot of time to do this, as my experience is scarce and this one's a bit different from the previous similar commits.

Reply:
that foreach is the while loop below `  while (*temp_storage) {`, so instead of having a printf, just call pj_s(pj, temp_cmd); to add a string to the previously opened array (pj_ka)